src/ex6/client/src/App.js

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import { ItemClient } from "./services/item_client";
+import TodoList from "./components/TodoList/TodoList";
+import TodoInput from "./components/TodoInput/TodoInput";
+import Header from "./components/Header/Header";
+import { Loader } from "./components/Loader/Loader";
+import "./App.css";
+import "monday-ui-react-core/dist/main.css";
+
+function App() {
+  const [todos, setTodos] = useState([]);
+  let itemClient = new ItemClient();
+
+  useEffect(() => {
+    const getTodos = async () => {
```

Comment:
why declaring the function inside the useEffect? decalre it outside and use in useEffect